draft-ietf-quic-qpack.md

```diff
@@ -279,27 +279,25 @@ table entries which have been acknowledged, but this could mean using
 literals. Since literals make the header block larger, this can result in the
 encoder becoming blocked on congestion or flow control limits.
 
-### Known Received Count
 
-The Known Received Count is the total number of dynamic table insertions and
-duplications acknowledged by the decoder.  The encoder tracks the Known Received
-Count in order to identify which dynamic table entries can be referenced without
-potentially blocking a stream.  The decoder tracks the Known Received Count in
-order to be able to send Insert Count Increment instructions (see
-{{insert-count-increment}}).
+### Known Received Count
 
-If a header block was potentially blocking, the encoder infers from receiving a
-Header Acknowledgement instruction ({{header-acknowledgement}}) that the decoder
-has received all dynamic table state necessary to process that header block.  If
-the Required Insert Count of the acknowledged header block is greater than the
-current Known Received Count, the encoder updates the Known Received Count to
-the value of the Required Insert Count of the acknowledged header block.
+The Known Received Count is the total number of acknowledged dynamic table
+insertions and duplications.  It is updated using Header Acknowledgement
```

Comment:
I'm not sure it was wrong before to say it's the number acknowledged by the decoder.   The decoder updates as it acknowledges, the encoder updates as it receives acknowledgements.  Without it, you need to read further to clarify that this is not the insert+dup count as sent by the encoder.